app/helpers/movies_helper.rb

```diff
@@ -0,0 +1,43 @@
+module MoviesHelper
+  require 'open-uri'
+  def average_review_score
+    (@movie.reviews.reduce(0){|score, r| score + r.rating } / @movie.reviews.length.to_f).round(1)
```

Comment:
Use Active Record calculation methods here, so that you are utilizing SQL to do the calculations, which is much faster than Ruby Enumerable methods. For example:
```ruby
@movies.review.sum(:rating)
```
Docs [here](http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html).